controls/treeview/control-template.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Control Template
+page_title: .NET MAUI TreeView Documentation - Control Template
+description: Review what is the default control template of the Telerik .NET MAUI TreeView control and all elements we have in the template. 
+position: 7
+slug: treeview-control-template
+---
+
+# .NET MAUI TreeView Control Template
+
+The Telerik UI for .NET MAUI TreeView control has a set of elements inside the `ControlTemplate`. These elements are `CheckBox`, `Image`, `Expand/Collapse` indicator and the `Label`. 
+
+If you want to fully customize the appearance of these elements you have to use the `ControlTemplate`. 
+
+Here is an example how to use the `ControlTemplate`. 
```

Comment:
Describe in more detail what the example shows. Start the sentence with:
"The following example shows how to do thing1, thing2, thing3 by using the `ControlTemplate`"